ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -7,6 +8,9 @@ import NetworkGraph from './NetworkGraph';
 import './NetworkGraphPage.css';
 
 function NetworkGraphPage() {
+    const { detailType, detailId } = useParams();
```

Comment:
Seems like we're not doing anything with the `detailType` and `detailId`. Maybe that's intentional and you were going to do some iterative work. I think to solve the issues highlighted above, we could have a custom hook that does the following:

1. Uses the `useParam` hook to get the `type` and `id` of a selected object (deployment, namespace, etc.)
2. Provides a function that can be used in the `onSelect` of the `NetworkGraph` component to update the URL

We can then pass the `type` and `id` as props to the `NetworkGraph` component and it can be used to determine what's selected

```
example:

const [selectedObject, setSelectedObject] = useNetworkObjectSelection();
...
return (
    ...
    <NetworkGraph selectedType={selectedObject?.type} selectedId={selectedObject?.id} />
    ...
)
```

Reply:
Using those params will be in part 2, _URL Network Graph_: Return to the URL_ ;)